package.sh

```diff
@@ -170,18 +181,34 @@ make_dir_tree() {
     fi
 }
 
-
 # do_build builds the code. The version and commit must be passed in.
 do_build() {
     for b in ${BINS[*]}; do
         rm -f $GOPATH_INSTALL/bin/$b
     done
+    
+    if [ ! -z "$WORKING_DIR" ]; then
```

Comment:
I add this to all modification of build process for unofficially supported platform that might need modification of packaging process. Since it needs an explicit switch and doesn't do anything more than packaging, it seemed ok to me. That was along my suggestion in #3755.